src/emulator/downloadableEmulators.ts

```diff
@@ -500,10 +500,15 @@ async function _runBinary(
           "Unsupported java version, make sure java --version reports 1.8 or higher.",
         );
       }
+
+      if (data.toString().includes("address already in use")) {
+        const message = `${description} has exited because its configured port ${command.port} is already in use. Are you running another copy of the emulator suite?`;
+        logger.logLabeled("ERROR", emulator.name, message);
+      }
     });
 
-    emulator.instance.on("error", (err) => {
-      handleEmulatorProcessError(emulator.name, err, command.port);
+    emulator.instance.on("error", async (err) => {
+      await handleEmulatorProcessError(emulator.name, err, command.port);
```

Comment:
I don't think this does anything. Event listener callbacks are not monitoring Promise returns and a rejection or a completion won't have visible effects.

To communicate this better, you'd probably want the following instead (this promise is not being handled):

```suggestion
      void handleEmulatorProcessError(emulator.name, err, command.port);
```